ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/TestXARecoveryHelper.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Couldn't this be in ArjunaJTS/jtax/tests/classes/com/hp/mwtests/ts/jta/jts/recovery/